src/rules/selector-max-specificity/README.md

```diff
@@ -0,0 +1,29 @@
+# selector-max-specificity
```

Comment:
The README for every rule follows [a consistent format](http://stylelint.io/developer-guide/rules/#readme). Have a look at the [README for `max-line-length`](https://github.com/stylelint/stylelint/tree/master/src/rules/max-line-length) as that's very similar to your rule.